### PR TITLE
fix(expogo): migrate to secure store & expo linear gradient & remove …

### PR DESCRIPTION
**Pull request by guardian-001**
- Migration from react-native-mmkv to expo-secure-store
- Migration from react-native-linear-gradient to expo-linear-gradient
- Removal of react-native-avoid-softinput

## Note !! : 
All these migrations and removals are due to the lack of support for certain React Native libraries in Expo Go, which require access to native code.

### CODE REVIEW

File: src/translations/i18n/utils.tsx [hunk 1]
```diff
@@ -15,7 +14,10 @@ export type TxKeyPath = RecursiveKeyOf<DefaultLocale>;
 
 export const LOCAL = 'local';
```
Comment (MohamedMegdich1) [inline src/translations/i18n/utils.tsx:15]: also this

**Review by MohamedMegdich1 — CHANGES_REQUESTED** [review]

File: src/translations/i18n/utils.tsx (hunk 1 above)
Comment (archimatchproject) [reply src/translations/i18n/utils.tsx]: fixed

File: src/shared/hooks/use-is-first-time.tsx [hunk 2]
```diff
@@ -1,13 +1,32 @@
-import { useMMKVBoolean } from 'react-native-mmkv';
-
-import { storage } from '../../core/storage';
+import * as SecureStore from 'expo-secure-store';
+import { useEffect, useState } from 'react';
 
 const IS_FIRST_TIME = 'IS_FIRST_TIME';
```
[the remark this answers is not in the document]
Comment (archimatchproject) [reply src/shared/hooks/use-is-first-time.tsx]: fixed

File: src/api/common/client.tsx [hunk 3]
```diff
@@ -7,8 +7,9 @@ export const client = axios.create({
 });
 client.interceptors.request.use(
   async (config) => {
-    const token = getToken();
+    const token = await getToken();
     if (token && token.access) {
+      console.log(token.access);
```
[the remark this answers is not in the document]
Comment (archimatchproject) [reply src/api/common/client.tsx]: removed

**Pull request merged**